### ISSUE TITLE
slf4j implementation (logback) preventing log4j usage

### ISSUE THREAD

**Issue by maxthomas** [NextCenturyCorporation/AIDA-Interchange-Format#226]
the way the `ValidateAIF` logger is set up is causing problems: it's being cast to a `logback` logger which is causing runtime exceptions for non-logback implementations:

```
Caused by: java.lang.ClassCastException: org.apache.logging.slf4j.Log4jLogger cannot be cast to ch.qos.logback.classic.Logger
	at com.ncc.aif.ValidateAIF.<clinit>(ValidateAIF.java:59)
```

(we use log4j but this would happen with any other)

the way i have typically seen slf4j loggers initialized is like this:

```
  private static final Logger LOG = LoggerFactory.getLogger(Demo.class); // or whatever modifiers
```

this uses the slf4j backend but allows clients to plug whichever implementation they want at runtime - which is what slf4j is designed to do, to some extent, abstract away the implementation and provide a logging framework. 

the way the current code is set up, the `logger` sets its own level to `info` to reduce verbosity (i'm guessing?) but this is perhaps better handled by adding configurations to make other noisy logging facilities quieter.

**Comment by nextcen-dgemoets** [comment]
Hi, Max, thank you for the report.  Although we haven't modified the way the Logger is instantiated and used since the Java version of `ValidateAIF` was created, we have recognized that it could use some improvement, especially as this class could be used programmatically by other developers.

To that end, In the master branch, the `ValidateAIF` class no longer contains a Logger.  We've separated the validator command-line tool out from the `ValidateAIF` class and put it into its own class, `ValidateAIFCli`.  It is still instantiated and used in `ValidateAIFCli` similarly to how it was before in `ValidateAIF`.

I'll write a ticket to fix this issue in the v1.0.x branch, as it impedes users of the `ValidateAIF` class.

**Comment by maxthomas** [comment]
sounds great - thanks!

**Comment by nextcen-dgemoets** [comment]
@maxthomas could you confirm that this is no longer a problem for you in the recently-released [AIF v1.0.4](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/tree/tag/v1.0.4)?  There is no logging in `ValidateAIF`; it is all in `ValidateAIFCli`.

**Comment by nextcen-dgemoets** [comment]
This issue is resolved in v1.0.4.
